### DISCUSSION TITLE
Triple Click

### DISCUSSION THREAD

**Discussion by sirmbhe** [sezanzeb/input-remapper#476] [Q&A] [answered]
Hey.

Thanks for the great software.

Does anyone knows on how to map a mouse button into a triple click left mouse button (usually it's being used to select entire paragraph).

I tried to use repeat(3, BTN_LEFT) but it isn't working. I've also tried other combination like BTN_LEFT+BTN_LEFT+BTN_LEFT but still not working.

Thank you for any help.

**Comment by ZeRenCZ** [comment, marked as answer]
try this:

`r(3,k(BTN_LEFT).w(100))`

'w' is wait time between clicks

**Reply by sezanzeb** [reply]
`repeat`, `key` and `wait` can be used nowadays instead of the short function names, for better readability

**Comment by sirmbhe** [comment]
That you so much! That's working flawlessly :)
